packages/ra-ui-materialui/src/form/TabbedForm.tsx

```diff
@@ -153,3 +156,12 @@ export const findTabsWithErrors = (children, errors) => {
         return acc;
     }, []);
 };
+
+const sanitizeRestProps = ({
+    record,
+    resource,
```

Comment:
Those two should be passed as we want to let users override them through props if needed and some child components call `useResourceContext` and `useRecirdContext`

Reply:
You are right, `resource` is still accessed in `TabbedFormView`. What about `record`, though?
I could not find any usage in TabbedFormView, wouldn't it make sense to not pass it then?

Reply:
I also don't quite understand why we are passing a bunch of unrelated props into `useResourceContext`, instead of just the `resource`? I guess this is for convenience? But at the same time it makes it much harder to follow the actual intent of the code and to see which props are actually needed and used there.